SC_redesign/Crop_and_Soil/soil/soil_data.py

```diff
@@ -30,16 +31,35 @@ class SoilData:
     previous_retention_parameter: Optional[float] = None
     """retention parameter for the previous day (mm) (used in SWAT 2:1.1.9)"""
     average_slope_fraction: float = 0.05
-    """average slope fraction of the subbasin"""
+    """average slope fraction of the subbasin (unitless)"""
     moisture_condition_parameter: Optional[float] = None
     """curve number value adjusted for moisture content (unitless) (SWAT 2:1.1.11)"""
 
+    # ---- percolation
+    vadose_zone_layer: Optional[LayerData] = None
+    """Datalayer object that represents the vadose zone, arbitrary top and bottom depths, starts with no water"""
+    time_step: float = 24
+    """length of time step over which percolatigit adon occurs (hours) """
```

Comment:
Is this what you meant?
```suggestion
    """length of time step over which percolation occurs (hours) """
```

Reply:
No, I will also update this